.github/workflows/create-release-pr.yml

```diff
@@ -0,0 +1,45 @@
+name: Create Release PR
+on:
```

Comment:
An option to run this manually too from the actions tab would probably be a good thing. Would also make testing a lot easier

Reply:
Yeah - also found this to test them locally: https://github.com/nektos/act